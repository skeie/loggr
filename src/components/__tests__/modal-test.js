import React from 'react';
import renderer from 'react-test-renderer';
import Modal from '../Modal';
test('Should render Modal correctly', () => {
    const tree = renderer.create(
        <Modal /> 
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
    