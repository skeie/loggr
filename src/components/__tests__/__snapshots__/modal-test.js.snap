exports[`test Should render Modal correctly 1`] = `
<Modal
  animationType="slide"
  onRequestClose={[Function]}
  style={
    Object {
      "backgroundColor": "blue",
    }
  }
  transparent={true}
  visible={true} />
`;
